host/host.go

```diff
@@ -155,3 +155,7 @@ func SensorsTemperatures() ([]TemperatureStat, error) {
 func timeSince(ts uint64) uint64 {
 	return uint64(time.Now().Unix()) - ts
 }
+
+func timeSinceMillis(ts uint64) uint64 {
+	return uint64(time.Now().UnixMilli()) - ts
+}

```

Comment:
Could you fix line endings?

Reply:
Thank you, I believe it is fixed now (host/host_windows.go also had wrong line endings so I fixed that too).